manifests/k/KDE/Kate/21.08.2/KDE.Kate.installer.yaml

```diff
@@ -1,14 +1,14 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
-PackageIdentifier: KDE.Kate
-PackageVersion: 21.08.2
-MinimumOSVersion: 10.0.0.0
-InstallerType: nullsoft
-Scope: machine
+PackageIdentifier: KDE.Kate.Release
+PackageVersion: master-1459
+MinimumOSVersion: 
+InstallerType: 
+Scope: 
 UpgradeBehavior: install
-Installers:
+Installers: 
 - Architecture: x64
-  InstallerUrl: https://binary-factory.kde.org/view/Windows%2064-bit/job/Kate_Release_win64/1456/artifact/kate-21.08.2-1456-windows-msvc2019_64-cl.exe
-  InstallerSha256: D95F20BE05A06358090201A956D608DD0457CC7916A159403F83E671D7F045A7
+  InstallerUrl: https://binary-factory.kde.org/view/Windows%2064-bit/job/Kate_Release_win64/lastSuccessfulBuild/artifact/kate-21.08.2-1459-windows-msvc2019_64-cl-sideload.appx
```

Comment:
This URL doesn't download the appx for me, it just takes me to the raw binary content and displays it in the browser

Reply:
http://github.com/microsoft/winget-pkgs/pull/33731#discussion_r741977143.

The .appx file is downloaded for me upon request via Firefox.